tests/env/typescript-node/tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+      "baseUrl": "",
+      "strict": true,
+      "lib": [
+          "esnext",
+          "dom" // Added to avoir Request type error
```

```suggestion
          "dom" // Added to get Request type error
```
Or maybe I don't know the work `avoir` in english

---

... 😅